homekit/controller/ble_impl/__init__.py

```diff
@@ -592,10 +592,15 @@ def find_characteristic_by_uuid(device, service_uuid, char_uuid):
         if ServicesTypes.get_short(service_uuid.upper()) == ServicesTypes.get_short(possible_service.uuid.upper()):
             service_found = possible_service
             break
-    logger.debug('searched service: %s', service_found)
+    logger.debug('searched service: %s', service_found.uuid)
```

Comment:
This will break if service_found remains none if no service was found?

Reply:
Indeed, fixed